components/statuslight/index.css

```diff
@@ -159,15 +158,14 @@
 
 	/* Dot */
 	&::before {
+		--spectrum-statuslight-spacing-computed-top-to-dot: calc(var(--mod-statuslight-spacing-top-to-dot, var(--spectrum-statuslight-spacing-top-to-dot)) - var(--mod-statuslight-spacing-top-to-label, var(--spectrum-statuslight-spacing-top-to-label)));
```

Comment:
I think this must have been a linter fix- I didn't intentionally move this and didn't see it until the PR was created.